Students/Bologa M. Marius - Vasile/Zoowsome/src/javasmmr/zoowsome/models/animals/Animals.java

```diff
@@ -0,0 +1,44 @@
+package javasmmr.zoowsome.models.animals;
+/**
+ * 
+ * @author  Marius Bologa
+ *
+ */
+public abstract class Animals {
```

Comment:
This should be `Animal`. You are modelling *one* animal not *many* animals in this class